Config/conftest.py

```diff
@@ -0,0 +1,39 @@
+from pytest import fixture
+from selenium import webdriver
+from Pages.base_page import BasePage
+from Pages.pages import Pages
+
+
+
+@fixture(autouse=True)
+def prepare_something():
+	print("Prepare something before test")
+	yield
+	print("Cleanup after test")
+
+
+@fixture(autouse=True, scope='session')
+def cleanup(driver):
+	yield driver
+	driver.close()
+
+
+@fixture(scope='session')
+def driver():
+	base_url = "https://www.ubereats.com/pl"
+	options = webdriver.ChromeOptions()
+	options.add_experimental_option("detach", True)
+	driver = webdriver.Chrome(options=options)
+	driver.get(base_url)
```

Comment:
Please, create another fixture for driver.get(base_url). It would be better to do that this way, as sometimes you'd like to use it directly in a test